src/server/pbsd_main.c

```diff
@@ -1784,6 +1779,16 @@ main(int argc, char **argv)
 	svr_save_db(&server);	/* final recording of server */
 	track_save(NULL);	/* save tracking data	     */
 
+	/* save any jobs that need saving */
+	for (pjob = (job *)GET_NEXT(svr_alljobs); pjob; pjob = (job *)GET_NEXT(pjob->ji_alljobs))
+		job_save_db(pjob);
+
+	/* save any reservations that need saving */
+	for (presv = (resc_resv *)GET_NEXT(svr_allresvs); presv; presv = (resc_resv *)GET_NEXT(presv->ri_allresvs))
+		resv_save_db(presv);
+
+	save_nodes_db(0, NULL);
```

Comment:
Do we need to save any other object like a queue or server?

Reply:
This is basically a old code block before my attribute refactoring change. I had missed to pull in this block of code. The old code only saved these objects, so i just brought back these. Since it has survived so long without saving anything else, i guess we have set expectations here.